Formula/conduit.rb

```diff
@@ -0,0 +1,45 @@
+class Conduit < Formula
+  desc "Streams data between data stores. Kafka Connect replacement. No JVM required"
+  homepage "https://conduit.io/"
+  url "https://github.com/ConduitIO/conduit/archive/refs/tags/v0.5.0.tar.gz"
+  sha256 "ba57506c17a99356c443d3c4459977c825dcce1b039965ba7699140e11d95afc"
+  license "Apache-2.0"
+  head "https://github.com/ConduitIO/conduit.git", branch: "main"
+
+  depends_on "go"
+  depends_on "node"
+  depends_on "yarn"
+
+  def install
+    inreplace "Makefile" do |s|
+      s.change_make_var! "VERSION", version.to_s
+    end
+    system "make"
+    bin.install "conduit"
+  end
+
+  test do
+    # Assert conduit version
+    assert_match(version.to_s, shell_output("#{bin}/conduit -version"))
+
+    File.open("output.txt", "w") do |file|
```

Comment:
Is there no command line argument to direct the log somewhere?

Reply:
Unfortunately not

Reply:
How about `cmd --something > file.log`?

Reply:
@SMillerDev I tried that, but running `brew audit --strict --new-formula --online conduit` said that I should use a file block instead

Reply:
And did you try `shell_output("cmd --something")` already?

Reply:
yes, `shell_output` was my first approach, it didn't work cuz I needed to kill the process to get the logs from it. So I went with `open3` to kill the process, then changed it to `io.popen`, then redirected the output to the file as you suggested.